sca/windows/cis_win2012r2_domainL1_rcl.yml

```diff
@@ -20,6 +20,13 @@ policy:
   references:
     - https://www.cisecurity.org/cis-benchmarks/
 
+requirements:
+  title: "Check that the Windows platform is Windows Server 2012 R2"
+  description: "Requirements for running the audit policy under a Windows platform"
```

Comment:
These are not the audit policies -> `Requirements for running the CIS benchmark Domain Controller L1 under Windows Server 2012 R2`.

The same for others.